### PR TITLE
tidb_binlog/readme: update wording of binlogctl readme

### PR DESCRIPTION
**Pull request by CaitinChen**
@GregoryIan PTAL

### CODE REVIEW

File: tidb_binlog/binlogctl/README.md [hunk 1]
```diff
@@ -12,19 +12,20 @@ Usage of binlogctl:
...
+		Path of file that contains the list of trusted SSL CAs for connection with cluster components
 	-ssl-cert string
-		Path of file that contains X509 certificate in PEM format for connection with cluster components.
+		Path of file that contains X509 certificate in PEM format for connection with cluster components
 	-ssl-key string
-		Path of file that contains X509 key in PEM format for connection with cluster components.
+		Path of file that contains X509 key in PEM format for connection with cluster components
 	-time-zone Asia/Shanghai
-		set time zone if you want save time info in savepoint file, for example Asia/Shanghai for CST time, `Local` for local time
+		set time zone if you want to save time info in the savepoint file, for example `Asia`/`Shanghai` for CST time and `Local` for the local time
```
Comment (WangXiangUSTC) [inline tidb_binlog/binlogctl/README.md]: `Asia`/`Shanghai`  should changed to `Asia/Shanghai`,  Asia/Shanghai is a value of time zone

Comment (IANTHEREAL) [reply tidb_binlog/binlogctl/README.md]: please fix it @CaitinChen

**Comment by IANTHEREAL** [comment]
Rest LGTM

**Comment by IANTHEREAL** [comment]
@kennytm PTAL

File: tidb_binlog/binlogctl/README.md [hunk 2]
```diff
@@ -41,39 +42,49 @@ cd tidb-tools-latest-linux-amd64
...
-It would also generate a `{data-dir}/savepoint` meta file
+
+It will also generate a `{data-dir}/savepoint` meta file
+
+TODO: improve `meta` later, like adding offset of the Kafka topic that corresponds to each Pump node
 
 ## License
-Apache 2.0 license. See the [LICENSE](../LICENSE) file for details.
+
+Apache 2.0 license. See the [LICENSE](../LICENSE) file for details.
```
Comment (kennytm) [inline tidb_binlog/binlogctl/README.md]: Keep the trailing new line?

File: tidb_binlog/binlogctl/README.md [hunk 3]
```diff
@@ -41,39 +42,49 @@ cd tidb-tools-latest-linux-amd64
 
 ## Example
 
+### Query Pump/Drainer
 
+Run the following command:
 
-### query pumps/drainers
 ```
 bin/binlogctl -pd-urls=http://127.0.0.1:2379 -cmd pumps/drainers
```
Comment (kennytm) [inline tidb_binlog/binlogctl/README.md]: I suppose this means "`-cmd pumps` or `-cmd drainers`", not literally "`-cmd pumps/drainers`" right? If former, could we clarify e.g. by writing them in separate commands?

```sh
bin/binlogctl -pd-urls=http://127.0.0.1:2379 -cmd pumps
bin/binlogctl -pd-urls=http://127.0.0.1:2379 -cmd drainers
```

File: tidb_binlog/binlogctl/README.md [hunk 4]
```diff
@@ -41,39 +42,49 @@ cd tidb-tools-latest-linux-amd64
...
 ```
 2018/06/21 11:24:10 nodes.go:53: [info] pump: &{NodeID:ip-192-168-199-118:8250 Host:127.0.0.1:8250 IsAlive:true IsOffline:false LatestFilePos:{Suffix:0 Offset:15320} LatestKafkaPos:{Suffix:0 Offset:382} OfflineTS:0}
 ```
-we would format output later :)
 
+### Unregister Pump/Drainer
 
-### unregister pump/drainer
 ```
 bin/binlogctl -pd-urls=http://127.0.0.1:2379 -cmd delete-pump/delete-drainer -node-id ip-127-0-0-1:8250/{nodeID}
```
Comment (kennytm) [inline tidb_binlog/binlogctl/README.md]: (Ditto)

File: tidb_binlog/binlogctl/README.md (hunk 3 above)
Comment (CaitinChen) [reply tidb_binlog/binlogctl/README.md]: @GregoryIan ?

Comment (IANTHEREAL) [reply tidb_binlog/binlogctl/README.md]: yep

**Comment by CaitinChen** [comment]
@kennytm PTAL again. Thanks.

**Comment by kennytm** [comment]
LGTM

**Comment by IANTHEREAL** [comment]
thanks @CaitinChen

**Comment by CaitinChen** [comment]
@GregoryIan My pleasure~

**Pull request merged**